Makefile

```diff
@@ -65,12 +65,8 @@ build-integration: ## Build integration test binary
 
 .PHONY: test-e2e
 test-e2e: ## Run openshift specific e2e test
-	go test -timeout 60m \
-		-v ./vendor/github.com/openshift/cluster-api-actuator-pkg/pkg/e2e \
-		-kubeconfig $${KUBECONFIG:-~/.kube/config} \
-		-machine-api-namespace $${NAMESPACE:-openshift-machine-api} \
-		-ginkgo.v \
-		-args -v 5 -logtostderr true
+	cd ./vendor/github.com/openshift/cluster-api-actuator-pkg && \
```

Comment:
Prefer `$(MAKE) -C <dir>`.

Reply:
This didn't work.  https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_machine-api-operator/298/pull-ci-openshift-machine-api-operator-master-e2e-aws-operator/915